test/libyul/yulOptimizerTests/expressionSimplifier/exp_simplifications.yul

```diff
@@ -0,0 +1,22 @@
+{
+    let t := calldataload(0)
+    sstore(0, exp(0, t))
+    sstore(1, exp(1, t))
+    sstore(2, exp(2, t))
+    // The following should not be simplified
+    sstore(3, exp(8, t))
+}
+// ====
+// EVMVersion: >=byzantium
+// ----
+// step: expressionSimplifier
+//
+// {
+//     let _1 := 0
+//     let t := calldataload(_1)
+//     sstore(_1, iszero(t))
+//     sstore(1, 1)
+//     let _8 := 2
+//     sstore(_8, shl(t, 1))
```

Comment:
Note to self: This is not happening. What is happening is `sstore(_8, exp(_8, t))`

Reply:
The test failure only happens on the two ubuntu soltest. The rest of the soltest is working fine. Also locally it seems to be fine.